magento/pwa-studio
# PWA-2375 Improved version banner

## Description

Adding information about the installed PWA Studio packages to the build output. This will make it easier to determine what versions production sites are running for debugging and statistical purposes.

## Related Issue







Closes PWA-2375

## Acceptance





### Verification Stakeholders

@supernova-at 

### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature

1.  Run build (or skip to step 2 if verifying on PR instance)
2. Verify comment header in dist JS files (ex the client or vendors JS files)
3. Should contain information about the `@magento` and `@apollo` packages that are in use

## Checklist
